src/qtism/data/storage/xml/schemes/qticompact_v1p0.xsd

```diff
@@ -23,6 +23,8 @@
 	    			<xs:attribute name="adaptive" default="false" use="optional" type="xs:boolean"/>
 	    			<xs:attribute name="timeDependent" use="required" type="xs:boolean"/>
 	    			<xs:attribute name="endAttemptIdentifiers" use="optional" type="xs:string"/>
+                    <xs:attribute name="title" use="optional" type="xs:string"/>
```

Comment:
indentation issue

Reply:
Actually I have some indentation issues in this file. I would like to solve them after merging the PR so that we still see what is the exact modification. If not, the change will be overwhelmed between lot's of indentation fixes and will be difficult to track.

OK for you ?

Reply:
deal